css/style.css

```diff
@@ -158,26 +171,35 @@ li {
   margin: auto;
 }
 
+.projectpic1 {
+  width: 327px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
 .work1title {
   color: white;
   font-weight: 500;
   font-size: 32px;
   text-align: center;
   padding-top: 25px;
+  width: 250px;
 }
 
 .workdescription {
   color: white;
   font-size: 20px;
   font-family: 'inter', sans-serif;
   margin: 20px;
+  width: 300px;
 }
 
 .usedcode {
   display: flex;
   flex-direction: row;
   gap: 30px;
   align-items: center;
+  width: 300px;
 }
 
 button {
```

Comment:
- [ ] Please add styling for the other states of all the buttons (`active`, `hover` and `disabled`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design: 😁

![image](https://user-images.githubusercontent.com/20567503/177863092-64cedf59-5ff8-4a2f-b46d-c8568a66c002.png)